### ISSUE TITLE
Is it possible to create a bar chart with different colored columns?

### ISSUE THREAD

**Issue by chrisco484** [pingunaut/wicket-chartjs#35]
BTW I'm loving this library - it's so easy to use and when you do a Wicket AJAX update the charts re-animate like they're supposed to without me entering the el-weirdo world of JavaScript animation!

I've looked at the source code and BarDataSet's base class seems to only have a single set of colors that apply to all columns.

I've seen raw Chart JS bar charts that do have different colored columns. Is there a way to achieve this through Wicket Chart JS?

**Comment by chrisco484** [comment]
Ah... unless we're supposed to add multiple DataSets... yeah that probably makes sense! :)

**Comment by pingunaut** [comment]
@chrisco484 Thanks a lot first of all! So, is using multiple data sets an option that does solve your issue?

**Comment by chrisco484** [comment]
@pingunaut  Yes, I ended up using multiple datasets to get the different colors - but doing that is what made getting the Legend working important :)

My solution in the end was to just render my own legend in a simple HTML table that I place above the Bar chart in my main panel - works well but not perfect :)
